task_DEV_4/task_DEV_4/Parallelogram.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace task_DEV_4
+{
+    class Parallelogram : Square
+    {
+        public double Height { get; set; }
+        public double Angle { get; set; }
+        public double Side2 { get; set; }
```

Comment:
We need to avoid numbers in variable names.

Reply:
Done